<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="4.84s"
		durationms="4843"
		type="root"
		rid="34e34315b8ba2ba5"
		totalerrorcount="1"
		totalwarningcount="1"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="dev10"
		rxversion="9.2.1.0"
		host="VINICIUS-TESTER"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="pt-BR"
		dtnlanguage="pt-BR"
		screenresolution="1366x1536"
		timestamp="07/11/2022 09:19:48"
		timestampiso="2022-11-07T12:19:48"
		endtime="07/11/2022 09:19:52"
		timeoutfactor="1"
		runid=""
		sutversion=""
		module="CloseAUT">
		<detail>
			CloseAUT
		</detail>
		<errormessage>
			The process with id '-1' does not exist.
		</errormessage>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:03.549"
			timeRelativeToTestModuleStartTime="00:03.549"
			timeWallClock="09:19:51"
			time="00:03.549"
			level="Warn"
			category="Data"
			errimg="images_CloseAUT_707678a5\CloseAUT_707678a5_42f28985_trace.jpg"
			errthumb="images_CloseAUT_707678a5\CloseAUT_707678a5_42f28985_trace_thumb.jpg">
			<message>
				The following module variables are not bound to a data column:<br/>CloseAutProcessIDVar
			</message>
			<metainfo
				loglvl="Warn">
			</metainfo>
		</item>
		<item
			timeRelativeToTestSuiteStartTime="00:03.579"
			timeRelativeToTestModuleStartTime="00:03.579"
			timeWallClock="09:19:51"
			time="00:03.579"
			level="Info"
			category="Application"
			errimg="images_CloseAUT_707678a5\CloseAUT_707678a5_42f28985_trace.jpg"
			errthumb="images_CloseAUT_707678a5\CloseAUT_707678a5_42f28985_trace_thumb.jpg">
			<message>
				Closing application with Process ID bound to variable $CloseAutProcessIDVar.
			</message>
			<metainfo
				itemindex="0"
				codefile="c:\Users\dev10\Documents\Ranorex\RanorexStudio Projects\SomaGestao\SomaGestao\CloseAUT.cs"
				codeline="95"
				loglvl="Info">
			</metainfo>
		</item>
		<item
			timeRelativeToTestSuiteStartTime="00:04.766"
			timeRelativeToTestModuleStartTime="00:04.766"
			timeWallClock="09:19:52"
			time="00:04.766"
			level="Error"
			category="Module"
			errimg="images_CloseAUT_707678a5\CloseAUT_707678a5_e8d6102b_trace.jpg"
			errthumb="images_CloseAUT_707678a5\CloseAUT_707678a5_e8d6102b_trace_thumb.jpg">
			<message>
				The process with id '-1' does not exist.<br/>Processo com identifica&#231;&#227;o -1 n&#227;o est&#225; em execu&#231;&#227;o.
			</message>
			<metainfo
				stacktrace="   em Ranorex.Host.KillApplication(Int32 processId)
   em Ranorex.Host.CloseApplication(Int32 processId, Duration gracePeriod)
   em SomaGestao.CloseAUT.Ranorex.Core.Testing.ITestModule.Run() na c:\Users\dev10\Documents\Ranorex\RanorexStudio Projects\SomaGestao\SomaGestao\CloseAUT.cs:linha 96
   em Ranorex.Core.Testing.TestModuleLeaf.RunInternal(DataContext parentDataContext, Int32 iteration, Int32 iterationCount, Boolean skipIteration)"
				codefile="c:\Users\dev10\Documents\Ranorex\RanorexStudio Projects\SomaGestao\SomaGestao\CloseAUT.cs"
				codeline="96"
				loglvl="Error"
				itemindex="0">
			</metainfo>
		</item>
	</activity>
</report>
